packages/core/theme/src/components/toggle.ts

```diff
@@ -100,7 +100,7 @@ const toggle = tv({
     },
     size: {
       sm: {
-        wrapper: "w-10 h-6 mr-2 rtl:ml-2 rtl:mr-[unset]",
+        wrapper: "w-10 h-6 rtl:ml-2 rtl:mr-[unset]",
```

Comment:
Maybe if we are going forward with this. We should also fix the left margin in case of `rtl`.
<img width="183" alt="Screenshot 2024-10-10 at 12 07 12 PM" src="https://github.com/user-attachments/assets/760260cd-0f75-4ebf-b3be-505241182890">

Reply:
also please don't use `rtl:`. we're trying to get rid of it. ref: https://tailwindcss.com/blog/tailwindcss-v3-3#simplified-rtl-support-with-logical-properties

Reply:
Ack, noted. Thanks for the blog link.
It would make the styles much more clean imo with logical properties.